### ISSUE TITLE
Add ability to install extensions alongside flask for testing conflicts

### ISSUE THREAD

**Issue by meandavejustice** [hugomrdias/filsnap-testing#25]
Hey @hugomrdias, I'm working on a test for https://github.com/filecoin-project/filsnap/issues/160, looks like cleanest way to accomplish this would be to modify `metamask-testing-tools` to allow for installation of rainbow &|| OKX Wallet extensions, maybe this can passed in a key to `createFixture`

Initial thoughts, something like: 
```
let fixture = createFixture({
  download: {
    flask: true,
   additionalExtensions: [{
    repo: 'rainbow-me/browser-extension'
  }]
  },
})
```

I'll be working on this sometime this week, just wanted to start a discussion to see if you had a strong preference not to include something like this

**Comment by hugomrdias** [comment]
i agree, that package is made for metamask, but should be easy to extract the extension setup code.
